checkstyle/import-control-core.xml

```diff
@@ -49,6 +49,7 @@
     <allow pkg="kafka.server"/>
     <allow pkg="kafka.tools"/>
     <allow pkg="org.apache.kafka.clients"/>
+    <allow pkg="org.apache.kafka.server"/>
```

Comment:
I don't think we need the changes in this file

Reply:
you right wit is not needed I think I forgot it while am breaking the main PR

Reply:
removed it now

Reply:
Thanks, I'll wait for the CI to run then I'll take another look.